mauro-data-mapper/Dockerfile

```diff
@@ -8,14 +8,19 @@ LABEL org.opencontainers.image.authors="Oliver Freeman <oliver.freeman@bdi.ox.ac
 ARG MDM_APPLICATION_COMMIT=main
 ARG MDM_UI_COMMIT=main
 ARG ADDITIONAL_PLUGINS=""
-ARG MDM_UI_THEME_NAME="default"
+ARG MDM_UI_THEME_NAME="nhs-digital"
 ARG MDM_UI_FEATURE_SUBSCRIBED_CATALOGUES="true"
 
 ARG DEPLOY_MDM_EXPLORER=0
 ARG MDM_EXPLORER_VERSION="1.0.0-SNAPSHOT"
 ARG MDM_EXPLORER_BUILD_HOME=/opt/mdm-explorer/dist
 ARG MDM_EXPLORER_API_ENDPOINT="http://localhost:${MDM_PORT}/api"
```

Comment:
Other references to MDM Explorer which could probably be removed (including anything else in the repo you spot which is related).

Reply:
All explorer references and sources removed